server/region.go

```diff
@@ -119,6 +119,15 @@ func (r *regionInfo) GetFollowers() map[uint64]*metapb.Peer {
 	return followers
 }
 
+func (r *regionInfo) GetFollower() *metapb.Peer {
```

Comment:
It's hard for PD to tell which follower has newest raft logs. Perhaps transfer to a random follower is not always the best, but certainly not worse than the original.